### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by pkuang5**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -60,6 +61,8 @@ class App extends Component {
             <Route path="/settings" exact strict component={() => <Settings googleId={this.state.googleId} />}></Route>
             <Route path={"/" + this.state.username} exact strict component={() => <Profile googleId={this.state.googleId} username={this.state.username} />}></Route>
             <Route path="/:username" exact strict render={({match})=><Profile username={match.params.username}/>}/>
+            <Route path="/:username/movies/:id" exact strict render={({match})=><Movie movieId={match.params.id} username={match.params.username} />}/>
+            <Route path="/settings" exact strict component={() => <Settings googleId={this.state.googleId} />}></Route>
```
Comment (nateelee) [inline src/App.js:64]: Why are there two route paths to "/settings"

File: src/components/movie.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+function Movie(props){
+
+    const [coverImage, setCoverImage] = useState('')
+    const [dateOfEntry, setDateOfEntry] = useState('')
+    const [description, setDescription] = useState('')
+    const [name, setName] = useState('')
+    const [rating, setRating] = useState('')
+
+    useEffect(() => {
```
Comment (nateelee) [inline src/components/movie.jsx:12]: where is useEffect() being called?

**Review by nateelee — CHANGES_REQUESTED** [review]
There are only two things that I need clarification on.

File: src/App.js (hunk 1 above)
Comment (pkuang5) [reply src/App.js:64]: good catch. ill delete one of them

File: src/components/movie.jsx (hunk 2 above)
Comment (pkuang5) [reply src/components/movie.jsx:12]: useEffect is the same as component did mount for functional components

Comment (nateelee) [reply src/components/movie.jsx:12]: got it

**Comment by nateelee** [comment]
LGTM

**Pull request merged**